Makefile.in

```diff
@@ -0,0 +1,65 @@
+#
+# Makefile rules for the meataxe64 package
+#
+KEXT_NAME = meataxe64
+KEXT_SOURCES = \
+   src/meataxe64.c \
+   src/functions.c \
+   src/slab.c \
+   src/bitstring.c \
+   src/mtx64/bitstring.c \
+   src/mtx64/conex.c \
+   src/mtx64/dfmtf.c \
+   src/mtx64/fech.c \
+   src/mtx64/ffn5.c \
+   src/mtx64/field.c \
+   src/mtx64/fmul.c \
+   src/mtx64/fpef.c \
+   src/mtx64/fphpmi.cc \
+   src/mtx64/fphpmi1.cc \
+   src/mtx64/ftra.c \
+   src/mtx64/funs1.c \
+   src/mtx64/funs2.c \
+   src/mtx64/funs3.c \
+   src/mtx64/funs4.c \
+   src/mtx64/funs5.c \
+   src/mtx64/gauss.c \
+   src/mtx64/hpmi.c \
+   src/mtx64/io.c \
+   src/mtx64/linf.c \
+   src/mtx64/mech.c \
+   src/mtx64/mmul.c \
+   src/mtx64/mpef.c \
+   src/mtx64/pcritc.c \
+   src/mtx64/pmul.c \
+   src/mtx64/proggies.c \
+   src/mtx64/slab.c \
+   src/mtx64/tabmake.c \
+   src/mtx64/tfarm1.c \
+   src/mtx64/tfarm2.c \
+   src/mtx64/util.c
+
+ifeq ($(shell uname), Darwin)
+KEXT_SOURCES += src/mtx64/osx-pc1.s \
+    src/mtx64/osx-pc2.s \
+    src/mtx64/osx-pc3.s \
+    src/mtx64/osx-pc5.s \
+    src/mtx64/osx-tfarm0.s
+else
+KEXT_SOURCES += src/mtx64/pc1.s \
+    src/mtx64/pc2.s \
+    src/mtx64/pc3.s \
+    src/mtx64/pc5.s \
+    src/mtx64/tfarm0.s
+endif
+
+KEXT_CXXFLAGS = -std=c++11
+
+# Note: asmfix generates all the files at once, so the following command is
+# not quite safe to use in a parallel make
+src/mtx64/osx-%.s : src/mtx64/%.s etc/asmfix.g
+	$(GAP) -A -q etc/asmfix.g
```

Comment:
It would be good to resolve the above note; also, gap is slow... Some ideas are listed on < https://stackoverflow.com/questions/1034852>; e.g. one could use the C preprocessor and define a `CDECL(sym)` macro which maps to either `sym` or `_sym`, depending on the target, and then use that macro in the assembler code for global labels.

Reply:
Rewriting the assembler code is not an option (it comes from upstream). Replacing the GAP script by an awk script or something would work.

Reply:
I found a way to redo this rule that avoids the parallel build issues completely :-).